src/ontology/mondo-edit.obo

```diff
@@ -223276,12 +223272,8 @@ xref: OMIM:603543 {source="MONDO:equivalentTo", source="Orphanet:69085", source=
 xref: Orphanet:69085 {source="OMIM:603543", source="MONDO:equivalentTo"}
 xref: SCTID:721972001 {source="MONDO:equivalentTo"}
 xref: UMLS:C1863753 {source="OMIM:603543", source="MONDO:ncbi_mim2gene_medline", source="MONDO:notFoundInDiseaseSubset", source="Orphanet:69085", source="Orphanet:69085/e"}
-is_a: MONDO:0015853 {source="Orphanet:69085"} ! deficient breast volume or number
-is_a: MONDO:0018234 ! dysostosis
-is_a: MONDO:0018454 {source="MONDO:0018455-obsoleted"} ! dysostosis of genetic origin
-is_a: MONDO:0019054 ! congenital limb malformation
+is_a: MONDO:0003847 {source="https://orcid.org/0000-0001-5208-3432"} ! hereditary disease
 is_a: MONDO:0020197 {source="Orphanet:69085"} ! EEC syndrome and related syndrome
-is_a: MONDO:0800090 {source="PMID:31633310"} ! ectrodactyly with and without other manifestations
```

Comment:
were these additional removals/additions (other than Line 223279) of `is_a` done by a curator and therefore intended?

Reply:
yep, intended.

Note, I didn't exclude the superclass 'ectrodactyly with and without other manifestations' because it is tagged for obsoletion and will get removed anyway later